app/src/main/AndroidManifest.xml

```diff
@@ -318,7 +318,9 @@
             android:name=".feature.genericauth.otpinput.OtpInputActivity"
             android:exported="true"
             android:windowSoftInputMode="adjustResize" />
-        <activity android:name=".feature.claimdetail.ClaimDetailActivity" />
+        <activity
+            android:name=".feature.claimdetail.ClaimDetailActivity"
+            android:configChanges="colorMode|density|fontScale|fontWeightAdjustment|keyboard|keyboardHidden|layoutDirection|locale|mcc|mnc|navigation|orientation|screenLayout|screenSize|smallestScreenSize|touchscreen|uiMode" />
```

Comment:
Context: https://kotlinlang.slack.com/archives/CJLTWPH7S/p1641547722054300

Feel free to ask me to remove it if you don't feel comfortable doing it before official documentation comes for it though. I haven't found any issues with it myself